docs/send-an-email-when-a-products-price-goes-below-its-cost/README.md

```diff
@@ -28,10 +28,12 @@ shopify/products/update
 
 ## Documentation
 
-This task watches for product updates, and sends an email for each variant that's found to have a unit cost that's greater than the variant's for-sale price.
+This task watches for product updates, and sends an email (to the recipient of your choice) for each variant that's found to have a unit cost that's greater than the variant's for-sale price.
+
+It will only send one email per variant, per the last observed change to that specific variant cost and price pairing where the cost was greater than the price.
```

Comment:
> It will only send one email per variant, per the last observed change to that specific variant cost and price pairing where the cost was greater than the price.

Let's drop this one sentence, the extra detail will confuse some merchants